src/test/fuzz/feefrac.cpp

```diff
@@ -223,7 +223,7 @@ FUZZ_TARGET(feefrac_mul_div)
         if (mul64 < std::numeric_limits<int64_t>::max() / 1000 &&
             mul64 > std::numeric_limits<int64_t>::min() / 1000 &&
             quot_abs < arith_uint256{std::numeric_limits<int64_t>::max() / 1000}) {
-            CFeeRate feerate(mul64, (uint32_t)div);
+            CFeeRate feerate(mul64, (int32_t)div);
```

Comment:
you can remove the type conversion, I think it is no-op because div is implicitly `int32_t`